bindings/core/src/message_handler/account_handle.rs

```diff
@@ -0,0 +1,705 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(feature = "participation")]
+use std::str::FromStr;
+
+use iota_sdk::{
+    client::{
+        api::{PreparedTransactionData, PreparedTransactionDataDto, SignedTransactionData, SignedTransactionDataDto},
+        request_funds_from_faucet,
+    },
+    types::block::{
+        output::{
+            dto::{OutputBuilderAmountDto, OutputDto},
+            AliasId, AliasOutput, BasicOutput, FoundryOutput, NftId, NftOutput, Output, Rent, TokenId,
+        },
+        DtoError,
+    },
+    wallet::{
+        account::{
+            types::{AccountBalanceDto, TransactionDto},
+            AccountHandle, AliasOutputOptions, MintTokenTransactionDto, OutputDataDto, OutputOptions,
+            TransactionOptions,
+        },
+        message_interface::AddressWithUnspentOutputsDto,
+        AddressWithAmount, AddressWithMicroAmount, IncreaseNativeTokenSupplyOptions, NativeTokenOptions, NftOptions,
+    },
+};
+use primitive_types::U256;
+
+use super::Result;
+use crate::{method::AccountMethod, panic::convert_async_panics, Response};
+
+pub async fn call_account_method(account_handle: &AccountHandle, method: AccountMethod) -> Result<Response> {
```

Comment:
I think I would try to make it look like `call_client_method` and `call_wallet_method`, it's a bit inconsistent atm

Reply:
Well because they're currently different, account methods are only called over a WalletMethod
We can change this, but this also needs a change in all bindings then

Reply:
We can revisit later, but I don't like the duplication of the async panic catch stuff, this is done better for the 2 other method handling

Reply:
Ah yeah this can be reduced, will add a TODO for now